src/internal/packager/pkgcfg/load.go

```diff
@@ -0,0 +1,27 @@
+// SPDX-License-Identifier: Apache-2.0
```

Comment:
nit: This is a simple pkg, we can merge everything together. Rename load.go to `pkgcfg.go`